aten/src/ATen/native/mkldnn/xpu/FusionUtils.cpp

```diff
@@ -0,0 +1,104 @@
+
+#include <ATen/native/mkldnn/xpu/FusionUtils.h>
+
+using namespace at::native::onednn;
+
+namespace at::native::xpu {
+
+#define ATTR_FUNC(NAME)                                                       \
```

Comment:
Do we really need this to make code unreadable?

Reply:
Thanks for the suggestion! I would remove such marco. Agree that this would hinder the readability to user and its unneeded to add a maro this small utils.